src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Tab } from '../Tab/Tab';
+import { validTab, validCallback } from '../../propValidation';
```

Comment:
в этом случае используй ссылку от корня
```suggestion
import { validTab, validCallback } from 'src/propValidation';
```